src/app/pages/listings/contents/listing-card/listing-card.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="listing-card relative">
+  <div class="listing-card-header relative">
+    <h2 class="listing-header-title">
+      <a href="">{{ listing.title }}</a>
+    </h2>
+    <time [attr.title]="listing.title" [attr.datetime]="listing.published">
+      {{ listing.bumped_at | date: "MMM d" }}
+    </time>
+    <div class="listing-tags n-m">
+      <a
+        class="listing-tag-link"
+        href=""
+        *ngFor="let tag of listing.tags; let i = index"
+      >
+        <span  class="listing-tag-prefix"
+          >#</span
+        >{{ tag }}
+      </a>
+    </div>
+    <div class="listing-dropdown">
+      <button class="btn listing-dropdown-btn">
+        <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 12a2 2 0 11-4 0 2 2 0 014 0zm7 0a2 2 0 11-4 0 2 2 0 014 0zm5 2a2 2 0 100-4 2 2 0 000 4z"></path></svg>
+      </button>
+    </div>
```

Comment:
This should only appear when we hover on a card

Reply:
Need to add some logics tha'ts why i left as it is. Can i comment it out now?

Reply:
No if it is incomplete it is okay. Just wanted to point out that this is left